distribution/sender/routes/sync-response-consumer.xml

```diff
@@ -0,0 +1,40 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <!-- TODO(luis.pinto): which errorHandler to use? -->
+    <route id="sync-response-consumer" errorHandlerRef="shutdownErrorHandler">
+        <from uri="activemq:openmrs.sync.{{db-sync.senderId}}?connectionFactory=activeMqConnFactory&amp;acknowledgementModeName=CLIENT_ACKNOWLEDGE&amp;messageListenerContainerFactory=customMessageListenerContainerFactory&amp;asyncStartListener=true" />
```

Comment:
Ok, I'm working on this.

Reply:
Implemented and pushed, but not tested on deployed environment. Since I had started the development of the EC-89 after create this pull request and I was almost done, I will finalize it and then test the two tickets together as they are linked, if it's ok.